test/controller.spec.js

```diff
@@ -131,7 +147,7 @@ describe('Controller API', function () {
       }
     })
 
-    describe('should apply config', () => {
+    describe.skip('should apply config', () => {
```

Comment:
shouldn't this be skipped for `rust-ipfs` only? same question for the other skip, could be that I am missing some conditionality for rust-ipfs.

Reply:
This skip, yes... the next skip though is because of failing Go tests

Reply:
Skipped these tests